draft-ietf-rats-corim.md

```diff
@@ -1756,40 +1777,45 @@ For example, if the Evidence format is known in advance, CoRIMs using a profile
 
 The selection process MUST yield at least one usable tag.
 
+Selected CoRIMs are transformed into an internal representation (see {{sec-phase1-trans}}).
+If CoBOMs are required by policy, the selected tags are considered *inactive*.
+An *inactive* tag MUST NOT be used during the Appraisal Procedure.
```

Comment:
I find this way of presenting activation/disactivation a bit puzzling.

I see the flow from undecided/unknown to either active or inactive.
It'd be probably easier to grok if we introduced an initial "unknown" state and describe the decision tree as:

```mermaid
flowchart TD
    A[unknown] --> C{use_cobom}
    C -->|true| D{is_listed}
    C -->|false| E[active]
    D -->|true| F[active]
    D -->|false| G[inactive]
```

Reply:
> A diagram like this would be helpful IMO.

But maybe the diagram should be backed out of this PR until the tooling issues can be addressed. It's difficult to suggest changes when it doesn't build.